RelationalAI/Rest.cs

```diff
@@ -41,21 +41,16 @@ public class Rest
 
         private readonly ILogger _logger;
 
-        public Rest(Context context, ILogger logger, IAccessTokenHandler accessTokenHandler)
+        public Rest(Context context, ILogger logger)
         {
             _context = context;
             HttpClient = new HttpClient();
             _logger = logger ?? new LoggerFactory().CreateLogger("RAI-SDK");
-
-            // Init AccessTokenHandler
-            AccessTokenHandler = accessTokenHandler ?? new DefaultAccessTokenHandler();
-            AccessTokenHandler.Rest = this;
-            AccessTokenHandler.Logger = _logger;
         }
 
         public HttpClient HttpClient { get; set; }
 
-        public IAccessTokenHandler AccessTokenHandler { get; set; }
+        internal IAccessTokenHandler AccessTokenHandler { get; set; }
```

Comment:
why does the assembly need access to this prop?

Reply:
Making the field internal so that the client class can set and manipulate the access token handler in the rest class